### ISSUE TITLE
Add possibility to customize tag used to include heading in TOC.

### ISSUE THREAD

**Issue by 3ndriu** [sethen/markdown-include#13]
It would be nice if there was possibility to customize tag (I mean `!heading`) which tells whether specified heading is included in table of contents.

I have a case, when I compile a few markdown files into a single file and I'd like to have a TOC in resulting file, but at the same time I'm using the original file and I don't want `!heading` text to be visible in headings. With customization I could put tag inside comment and specify that TOC should contain headings marked with, for instance, `<!-- !heading -->`.

**Comment by sethen** [comment]
I am afraid I am having a hard time understanding what you're asking here.  Could you give me a more concrete example of the functionality you would like to see here?

**Comment by 3ndriu** [comment]
In short, currently if I want to have some heading included in table of contents generated by markdown-include, I have to do it like this: `# Some heading !heading`. It would be nice to have the possibility to replace `!heading` with something else so I can write, for instance, `# Some heading !whatever` and have the `Some heading` entry in the table of contents.

I hope it's more clear now.

**Comment by sethen** [comment]
Oh, so you just want a way to be able to replace the keyword of `!heading` with something else that you've defined?  That seems easy enough.  Currently there is nothing like that in markdown-include for you to be able to define your own custom keywords for `!heading`.

I will look at this and see if it's something I can implement quickly.

**Comment by 3ndriu** [comment]
Yes, this is exactly what I was asking about. Thanks in advance.
